common/src/Microsoft.Azure.IIoT.Hub.Mock/src/Services/IoTHubServices.cs

```diff
@@ -113,7 +113,8 @@ public class IoTHubServices : IIoTHubTwinServices,
                 DeviceId = deviceId,
                 ModuleId = moduleId,
                 EnqueuedTimeUtc = DateTime.UtcNow,
-                Message = new Message(payload)
+                Message = new Message(payload),
+                MessageData = payload
```

Comment:
Why adding the same payload twice?

Reply:
This is as "Message = new Message(payload)" is disposed and we are unable to surface the payload at test level for verification.

Reply:
ok, because this is a mock and no production code